src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pynput.py

```diff
@@ -0,0 +1,14 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import collect_submodules
+hiddenimport = collect_submodules("pynput")
```

Comment:
It [fails testing](https://github.com/bwoodsend/pyinstaller-hooks-contrib/actions/runs/1103129291) because of this:
```suggestion
hiddenimports = collect_submodules("pynput")
```